src/com/seafile/seadroid2/ui/activity/BrowserActivity.java

```diff
@@ -1047,7 +1047,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
                 try {
                     if (!tempDir.exists()) {
```

Comment:
if you changed to 
```
File tempDir = new File(DataManager.getExternalTempDirectory(), "saf_temp" + "/" + "upload-"+System.currentTimeMillis());
```
the `tempDir.exists()` will always be false, and should be removed.